docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Hardware Key Support (YubiKey)
+description: Hardware Key Support
+---
+
+## Introduction
+
+By default, `tsh`, Teleport Connect, and other Teleport clients store a user's login session directly on their filesystem. If a user's filesystem is compromised, any of their active Teleport login sessions would also be compromised.
+
+[Per-session MFA](per-session-mfa.mdx) provides a way to require an MFA presence check with a user's login session, but this check is only used for starting sessions with Teleport services (SSH, Kube, DB, etc.). This means that non-session actions, like administrative commands run with `tctl`, can still be run with an exfiltrated login session. 
+
+To prevent these types of exfiltration attacks altogether, Teleport clients support hardware-based private keys. Unlike disk-based private keys, hardware-based private keys are generated and stored directly on a hardware device and are impossible to export. With this features, a user's login session is only functional with access to the user's hardware device.
+
+Additionally, this feature can be configured to require touch for every Teleport request, including non-session requests. With touch required, Hardware Key Support provides better security in comparison to our per-session MFA offering.
+
+<Admonition type="note" title="touch caching">
+
+  A user's touch is cached on the YubiKey for 15 seconds to prevent excessive touch prompts.
+
+</Admonition>
+
+<ScopedBlock scope="oss">
+
+  Hardware-backed Login Sessions require Teleport Enterprise or Teleport Cloud.
```

Comment:
Teleport Cloud _is_ Teleport Enterprise. Let's confirm with @xinding33 how we should phrase this.